### PR TITLE
Create three qene

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I am not content with the titles of LIT7060Qene (also no accusative in mǝśyāṭ) and LIT7062Qene, that do not make sense to me. I hope, you will find better solutions.

I am not experienced enough in Qene to identify the qene type with certainty. LIT7060Qene may be of Zayǝʾǝze-type as well, because it has 5 verses.

### CODE REVIEW

**Review by nafisa-valieva — COMMENTED** [review]
OK

File: new/LIT7061Qene.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55rc-va. Division into verses is indicated with
+                    <foreign xml:lang="gez">።</foreign>. The end of the poem is indicated with <foreign xml:lang="gez">። ።</foreign>.</note>
+                <ab>
+                    <l n="1">ድንግለ፡ 20ኤል፡ ናንክር፡ እምአግምሮትኪ፡ ወሠውሮትኪ፡ ቃለ፡ አማነ፡ ህላዊ፡ ስፉሕ።</l>
+                    <l n="2">በአማነ፡ ጽበት፡ ከርሥኪ፡ መንጦላዕተ፡ ምkwəራብ፡ ንጽሕ።</l>
+                    <l n="3">ኀበ፡ አግምሮሰ፡ ወሠወሮ፡ ውሁድ፡ ምሳሌ፡ በአማን፡ ብዙኅ።</l>
```
Comment (DenisNosnitsin1970) [inline new/LIT7061Qene.xml:63]: Please change this to fidel.

File: new/LIT7061Qene.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7061Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ድንግለ፡ እስራኤል፡ ናንክር፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Dǝngǝla ʾƎsrāʾel nānkǝr (Qǝne of the Mawaddǝs-type)</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT7061Qene.xml:9]: mawaddǝs type (let's stick to one standard).

File: new/LIT7061Qene.xml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                <title xml:lang="en" corresp="#t1">The Virgin of Israel, we admire …</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT7061Qene.xml:10]: "we should admire" (jussive). Or, to make a bit more sense out of it "We should admire the Virgin of Israel ..."

**Comment by DenisNosnitsin1970** [comment]
Thank you!

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Pull request merged**